eventing/samples/k8s-events/flow.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: flows.knative.dev/v1alpha1
+kind: Flow
+metadata:
+  name: k8s-event-flow
+  namespace: default
+spec:
+  serviceAccountName: feed-sa
+  trigger:
+    eventType: dev.knative.k8s.event
+    # This is kind of superfluous due to parameters, but the current
+    # feed model still requires a resource
+    # Future revisions may opt to have a meaningful resource string
+    # that allows users to scope which objects they choose based on
+    # a templatized self-link. E.g. Configurations across all namespaces:
+    # /apis/serving.knative.dev/v1alpha1/namespaces/{namespace}/configurations/{config}
```

Comment:
I'd probably remove all of these comment lines. They're distracting and won't have meaning to the new user.